plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/RocksDBKeyValueStorageFactory.java

```diff
@@ -60,20 +61,20 @@
   RocksDBKeyValueStorageFactory(
       final Supplier<RocksDBFactoryConfiguration> configuration,
       final List<SegmentIdentifier> segments,
-      final int DEFAULT_VERSION) {
+      final int defaultVersion) {
     this.configuration = configuration;
     this.segments = segments;
-    this.DEFAULT_VERSION = DEFAULT_VERSION;
+    this.defaultVersion = defaultVersion;
   }
 
   public RocksDBKeyValueStorageFactory(
       final Supplier<RocksDBFactoryConfiguration> configuration,
       final List<SegmentIdentifier> segments) {
-    this(
-        configuration,
-        segments,
-        /** Source of truth for the default database version. */
-        1);
+    this(configuration, segments, DEFAULT_VERSION);
+  }
+
+  public int getDefaultVersion() {
```

Comment:
Is this method still needed?

Reply:
(web-page search didn't find a match in use ...but that was only a `find` on the GitHub diff PR)

Reply:
it is required